reference/7.2/Microsoft.PowerShell.Core/About/about_Automatic_Variables.md

```diff
@@ -492,6 +492,69 @@ from the originating session. To add data to the **ApplicationArguments**
 property, use the **ApplicationArguments** parameter of the
 `New-PSSessionOption` cmdlet.
 
+### $PSStyle
+
+> [!NOTE]
+> This is an experimental feature. For more information see
+> [about_Experimental_Features](about_Experimental_Features.md).
+
+As of PowerShell 7.2 you can now access the `$PSStyle` automatic variable to
+view and change the rendering of ANSI string output. The variable contains the
+following properties:
+
+| Property        | Description                                                |
+| --------------- | ---------------------------------------------------------- |
+| AttributesOff   | |
```

Comment:
This was renamed to `Reset` which turns off all decoration (colors, styles, etc...)

Reply:
Much appreciated! Fixed